L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker.cc

```diff
@@ -499,6 +504,10 @@ class L1TrackObjectNtupleMaker : public edm::one::EDAnalyzer<edm::one::SharedRes
   std::vector<int>* m_trkjetem_ntracks;
   std::vector<int>* m_trkjetem_nxtracks;
 
+  std::vector<float>* m_triplet_phi;
+  std::vector<float>* m_triplet_eta;
+  std::vector<float>* m_triplet_pt;
```

Comment:
Why do all of these need to be pointers to vectors?

Reply:
Hello, 

I just followed the convention of the output for consistency. Would not be my first choice otherwise. I think this code will be deprecated soon and replaced by the L1T nano.

Best,
george